spec/errors_spec.rb

```diff
@@ -60,6 +60,11 @@ def execute
     assert_equal "Newsletter Subscription isn't a boolean", atom.message
   end
 
+  it "titleizes keys and keeps Id postfix" do
+    atom = Mutations::ErrorAtom.new(:newsletter_subscription_id, :boolean)
+    assert_equal "Newsletter Subscription Id isn't a boolean", atom.message
```

Comment:
I would expect the titleized version of "newsletter_subscription_id" to be "Newsletter Subscription ID".